locales/br.lua

```diff
@@ -1,238 +1,238 @@
+  ['weapon_compactlauncher'] = 'Lançador compacto',
+  ['weapon_minismg'] = 'mini SMG',
+  ['weapon_pipebomb'] = 'Bomba caseira',
+  ['weapon_poolcue'] = 'Taco de sinuca',
+  ['weapon_wrench'] = 'Chave de cano',
+  ['weapon_flashlight'] = 'Laterna',
+  ['gadget_parachute'] = 'Paraquedas',
   ['weapon_flare'] = 'flare',
-  ['weapon_doubleaction'] = 'double-Action Revolver',
-  ['weapon_pistol_mk2'] = 'pistol Mk2',                   -- needs translation
-  ['weapon_smg_mk2'] = 'SMG Mk2',                         -- needs translation
-  ['weapon_assaultrifle_mk2'] = "assault rifle Mk2",      -- needs translation
-  ['weapon_carbinerifle_mk2'] = 'carbine rifle Mk2',      -- needs translation
-  ['weapon_combatmg_mk2'] = 'combat MG Mk2',              -- needs translation
-  ['weapon_pumpshotgun_mk2'] = 'pump shotgun mk2',        -- needs translation
-  ['weapon_heavysniper_mk2'] = 'heavy sniper Mk2',        -- needs translation
-  ['weapon_snspistol_mk2'] = 'sns pistol Mk2',            -- needs translation
-  ['weapon_specialcarbine_mk2'] = 'special carbine Mk2',  -- needs translation
-  ['weapon_bullpuprifle_mk2'] = 'bullpup rifle Mk2',      -- needs translation
-  ['weapon_marksmanrifle_mk2'] = 'marksman rifle Mk2',    -- needs translation
-  ['weapon_revolver_mk2'] = 'heavy revolver Mk2',         -- needs translation
+  ['weapon_doubleaction'] = 'Revolver dupla-ação',
+  ['weapon_pistol_mk2'] = 'pistola Mk2',                   
+  ['weapon_smg_mk2'] = 'SMG Mk2',                         
+  ['weapon_assaultrifle_mk2'] = "Rifle de assalto Mk2",      
+  ['weapon_carbinerifle_mk2'] = 'carbina Mk2',      
+  ['weapon_combatmg_mk2'] = 'MG de combate Mk2',              
+  ['weapon_pumpshotgun_mk2'] = 'Espingarda mk2',        
+  ['weapon_heavysniper_mk2'] = 'Sniper pesada Mk2',        
+  ['weapon_snspistol_mk2'] = 'Pistola SNS Mk2',            
+  ['weapon_specialcarbine_mk2'] = 'Carabina especial Mk2',  
+  ['weapon_bullpuprifle_mk2'] = 'Bullpup rifle Mk2',      
+  ['weapon_marksmanrifle_mk2'] = 'Marksman rifle Mk2',    
+  ['weapon_revolver_mk2'] = 'Revolver pesado Mk2',         
 
   -- Weapon Components
-  ['component_clip_default'] = 'aderência padrão',
-  ['component_clip_extended'] = 'aderência prolongada',
-  ['component_clip_drum'] = 'drum Magazine',
-  ['component_clip_box'] = 'box Magazine',
-  ['component_flashlight'] = 'lanterna',
-  ['component_scope'] = 'mira',
-  ['component_scope_advanced'] = 'mira avançada',
-  ['component_suppressor'] = 'supressor',
-  ['component_grip'] = 'grip',
-  ['component_luxary_finish'] = 'acabamento de arma de luxo',
+  ['component_clip_default'] = 'Munição padrão',
+  ['component_clip_extended'] = 'Munição prolongada',
+  ['component_clip_drum'] = 'Munição de tambor',
+  ['component_clip_box'] = 'Caixa de munição',
+  ['component_flashlight'] = 'Lnterna',
+  ['component_scope'] = 'Mia',
+  ['component_scope_advanced'] = 'Mra avançada',
+  ['component_suppressor'] = 'Spressor',
+  ['component_grip'] = 'Pega
```

Comment:
Missing `',` here.

Reply:
Also realized some typos since my keyboard died that day... will fix it